Modules/Customer/Http/Services/CustomerService.php

```diff
@@ -117,18 +121,18 @@ public function update(array $requestData): array
      */
     public function getCustomers(Request $request): array
     {
-        $user = User::find($request->user_id);
+        $user = auth()->user();
         if ($user) {
             $brand = Brand::where('user_id', $user->id)->first();
-            $allCustomersCount = Customer::where('brand_id', $brand->user_id)->count();
-            $orderedCustomersCount = Customer::where('brand_id', $brand->user_id)->where('status', 'ordered')->count();
-            $contactedCustomersCount = Customer::where('brand_id', $brand->user_id)->where('status', 'contacted')->count();
-            $unusedCreditCustomersCount = Customer::where('brand_id', $brand->user_id)->where('status', 'unused credit')->count();
-            $notOrderedCustomersCount = Customer::where('brand_id', $brand->user_id)->where('status', 'not yet ordered')->count();
-            $uncontactedCustomersCount = Customer::where('brand_id', $brand->user_id)->where('status', 'uncontacted')->count();
-            $notSignedCustomersCount = Customer::where('brand_id', $brand->user_id)->where('status', 'not signed up')->count();
-            $onBazarCustomersCount = Customer::where('brand_id', $brand->user_id)->where('status', 'on bazar')->count();
-            $customers = Customer::where('brand_id', $brand->user_id);
+            $allCustomersCount = Customer::where('user_id', $brand->user_id)->count();
```

Comment:
@arijit1111 You can change it to the local scope as `Customer::auth()` In local scope the method should check for authorized user if the instance belongs to.

Reply:
@arijit1111 Aren't you going to make this change right now?

Reply:
These are updated in last push

Reply:
updated in latest push